app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.example.featuredetectandroid">
+
+    <application
+        android:allowBackup="true"
+        android:dataExtractionRules="@xml/data_extraction_rules"
+        android:fullBackupContent="@xml/backup_rules"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.FeatureDetectAndroid"
+        tools:targetApi="31">
+        <activity
+            android:name="me.alexdtat.featuredetectandroid.MainActivity"
+            android:exported="true"
+            android:label="@string/app_name"
```

Comment:
Android Studio tells me that this is redundant

Reply:
done